### ISSUE TITLE
Vue package version mismatch

### ISSUE THREAD

**Issue by cool-man-vk** [InternetHealthReport/ihr-website#195]

**Comment by sumana2001** [comment]
Yes, I am also facing the same error. When I am trying to run the application on localhost it shows me Vue mismatch due to which the application is not running. 

```
Vue packages version mismatch:

- vue@3.2.41 (/Users/sumana/Documents/ihr-website/node_modules/vue/index.js)
- vue-template-compiler@2.7.13 (/Users/sumana/Documents/ihr-website/node_modules/vue-template-compiler/package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the latest.
```

How do I solve this?

**Comment by thisiskaransgit** [comment]
Hey @cool-man-vk @sumana2001 
I faced the same issue, Stack overflow wasn't enough 

The following steps worked for me
1. remove the existing node_modules folder if you have created it
2. go to `Package.json`
3. Remove the caret sign(^), from the `vue` and `vue-template-compiler` versions
   "vue": "`^`2.6.11"   =>  "vue": "2.6.11"
    "vue-template-compiler": "`^`2.6.11" =>     "vue-template-compiler": "2.6.11"
  4. Run `npm install` && `npm run serve`
  
 Hope this helps !

**Comment by Aniket762** [comment]
Hey @thisiskaransgit, thanks for mentioning and answering the queries. It is surely of great help to users. 

@sumana2001 @cool-man-vk please let us know if this particular approach solves your concern. Looking forward to a positive response.

**Comment by sumana2001** [comment]
It worked! Thank you so much😊

**Comment by Aniket762** [comment]
Thanks a lot @sumana2001 for the feedback. 

@thisiskaransgit do you want to raise a PR with the fix? Incase you are busy, I would directly commit to the master. Thanks a lot for you help mate 💪

**Comment by cool-man-vk** [comment]
Thanks a lot guys! It worked for me

**Comment by thisiskaransgit** [comment]
hey @Aniket762, I have fixed the issue in the PR #186, sorry for not mentioning it before,I overlooked it,  If you want I'll create a fresh PR with the bug fix

**Comment by Aniket762** [comment]
> hey @Aniket762, I have fixed the issue in the PR #186, sorry for not mentioning it before,I overlooked it, If you want I'll create a fresh PR with the bug fix

A new PR would be great. Can you please do it
